src/main.zig

```diff
@@ -485,15 +501,19 @@ pub fn main() !void {
                         var sleep = std.ChildProcess.init(&[_][]const u8{ "/bin/sh", "-c", sleep_cmd }, allocator);
                         _ = sleep.spawnAndWait() catch .{};
                     }
-                } else if (pressed_key == 5) {
-                    if (c_import.access("/usr/bin/brightnessctl", c_import.X_OK) == 0) {
-                        var brightness = std.ChildProcess.init(&[_][]const u8{ "/usr/bin/brightnessctl", "-q", "s", "5%-" }, allocator);
-                        _ = brightness.spawnAndWait()catch.{};
+                } else if (pressed_key == brightness_down_key) {
+                    if (unistd.access(&config.brightnessctl[0], unistd.X_OK) == 0) {
+                        const brightness_str = std.fmt.allocPrint(allocator, "{s}%-", .{config.brightness_change}) catch unreachable;
+                        defer allocator.free(brightness_str);
+                        var brightness = std.ChildProcess.init(&[_][]const u8{ config.brightnessctl, "-q", "s", brightness_str }, allocator);
+                        _ = brightness.spawnAndWait() catch .{};
                     }
-                } else if (pressed_key == 6) {
-                    if (c_import.access("/usr/bin/brightnessctl", c_import.X_OK) == 0) {
-                        var brightness = std.ChildProcess.init(&[_][]const u8{ "/usr/bin/brightnessctl", "-q", "s", "+5%" }, allocator);
-                        _ = brightness.spawnAndWait()catch.{};
+                } else if (pressed_key == brightness_up_key) {
+                    if (unistd.access(&config.brightnessctl[0], unistd.X_OK) == 0) {
+                        const brightness_str = std.fmt.allocPrint(allocator, "+{s}%", .{config.brightness_change}) catch unreachable;
+                        defer allocator.free(brightness_str);
+                        var brightness = std.ChildProcess.init(&[_][]const u8{ config.brightnessctl, "-q", "s", brightness_str }, allocator);
+                        _ = brightness.spawnAndWait() catch .{};
                     }
                 }
```

Comment:
Instead of assuming the allocation will always succeed, it may be better instead to set the info line to the `err_alloc` error and break out of the inner condition, like so (same applies for the brightness down key, too):

```suggestion
                } else if (pressed_key == brightness_up_key and unistd.access(&config.brightnessctl[0], unistd.X_OK) == 0) brightness_change: {
                    const brightness_str = std.fmt.allocPrint(allocator, "+{s}%", .{config.brightness_change}) catch {
                        try info_line.setText(lang.err_alloc);
                        break :brightness_change;
                    };
                    defer allocator.free(brightness_str);
                    var brightness = std.ChildProcess.init(&[_][]const u8{ config.brightnessctl, "-q", "s", brightness_str }, allocator);
                    _ = brightness.spawnAndWait() catch .{};
                }
```

Reply:
Done.